### PR TITLE
Note to escape exclamation mark in *nix

### PR DESCRIPTION
**Pull request by joacar**
## Summary

Add note that exclamation mark need to be escaped on *nix.

~Fixes #Issue_Number (if available)~

### CODE REVIEW

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks for your contribution @joacar. I left a few suggestions.

@dsplaisted who can validate this change?

**Comment by joacar** [comment]
Thanks for correcting my errors.

**Comment by KathleenDollard** [comment]
@kendrahavens

**Comment by mairaw** [comment]
Can you commit my suggestions @joacar?

You can click on the button under the suggestion:
image

**Comment by joacar** [comment]
Sorry, I was certain I did. Will do it

**Comment by mairaw** [comment]
@joacar I still see my suggestions haven't been applied. Do you need any help doing that?

**Comment by joacar** [comment]
@mairaw Sorry for taking time but once again I thought I'd done that. Did it from mobile view so perhaps missed a "submit" button. Now I see the commit is pushed.

**Review by kendrahavens — APPROVED** [review]
Namespace is one word, but we can fix that in a different run. Thank you!

File: docs/core/testing/selective-unit-tests.md [hunk 1]
```diff
@@ -10,6 +10,11 @@ ms.custom: seodec18
 
 With the `dotnet test` command in .NET Core, you can use a filter expression to run selective tests. This article demonstrates how to filter which test are run. The following examples use `dotnet test`. If you're using `vstest.console.exe`, replace `--filter` with `--testcasefilter:`.
 
+> [!NOTE]
+> Using filters that include exclamation mark (!) on `*nix` requires escaping since `!` is reserved. For example, this filter
+> skips all tests if name space contains IntegrationTests `dotnet test --filter FullyQualifiedName\!~IntegrationTests`.
```
Comment (mairaw) [inline docs/core/testing/selective-unit-tests.md]:
```suggestion
> skips all tests if the namespace contains IntegrationTests: `dotnet test --filter FullyQualifiedName\!~IntegrationTests`.
```

**Review by mairaw — APPROVED** [review]
Thank you @joacar

**Pull request merged**